unattended-upgrade

```diff
@@ -1117,21 +1131,15 @@ def conffile_prompt(destFile, prefix=""):
     # now check if there are conffiles in the pkg that where not there
     # in the previous version in the dpkg status file
     if pkg_conffiles:
-        for conf_file in pkg_conffiles.split("\n"):
-            installed_conf_file = find_old_installed_conf_file(
-                conf_file, prefix)
-            if conf_file not in dpkg_status_conffiles \
-               and os.path.exists(installed_conf_file):
+        for conf_file in pkg_conffiles:
+            old_conf_file = map_conf_file(conf_file, dpkg_status_conffiles)
+            if not old_conf_file and os.path.exists(prefix + conf_file):
```

Comment:
Could we drop the `os.path.exists()` here, as the function `get_md5sum_for_file_installed` already handles that by returning "". Maybe adding `and current_md5 != ""` in the `if` below to make sure we don't get odd cases.

Reply:
Fixed that, too.